.gitignore

```diff
@@ -30,7 +30,10 @@ hcatalog/webhcat/svr/target
 conf/hive-default.xml.template
 itests/hive-blobstore/src/test/resources/blobstore-conf.xml
 .DS_Store
+.factorypath
 patchprocess
 standalone-metastore/src/gen/version
 standalone-metastore/metastore-common/src/gen/version
 standalone-metastore/metastore-server/src/gen/version
+launch.json
```

Comment:
are those a standard visual studio related files to exclude ?

Reply:
Yes. launch.json is defining program launch arguments and factorypath a temp directory for indexed code. Both are environment (VSCode) specific and might even contain absolute paths...